mantidproject/mantid
# Removed Mantid Plot from Mantid basics course

**Description of work.**

Both the menu entry for Mantid Plot and all files for the respective folder were removed.

**To test:**

Check the tutorial for the Mantid Basics Course. There should be no section for Mantid Plot anymore.

Fixes #31122 . 

*This does not require release notes* because it is only an update of the documentation of the Mantid Basics Course.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
